CHANGELOG.md

```diff
@@ -14,6 +14,22 @@ _This release is scheduled to be released on 2021-01-01._
 ### Updated
 
 - Weather module - forecast now show TODAY and TOMORROW instead of weekday, to make it easier to understand
+- Node packages:
```

Comment:
You can just write "update dependencies to latest version", no need for this detail :-)